index.html

```diff
@@ -0,0 +1,375 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="queries.css" />
+    <script
+      type="module"
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.esm.js"></script>
+    <script
+      nomodule
+      src="https://unpkg.com/ionicons@5.5.2/dist/ionicons/ionicons.js"></script>
+    <title>Online School</title>
+  </head>
+  <body>
+    <!-- main page header -->
+    <div class="container">
+      <header class="header sticky">
+        <nav class="navigation">
+          <ul class="top-nav">
+            <li class="top-nav--element">
+              <a href="#"
+                ><ion-icon class="facebook-icon" name="logo-facebook"></ion-icon
+              ></a>
+            </li>
+            <li class="top-nav--element">
+              <a href="#"
+                ><ion-icon class="twitter-icon" name="logo-twitter"></ion-icon
+              ></a>
+            </li>
+            <li class="top-nav--element"><a href="#">English</a></li>
+            <li class="top-nav--element"><a href="#">My Page</a></li>
+            <li class="top-nav--element"><a href="#">Logout</a></li>
+          </ul>
+          <a href="index.html">
+            <img
+              class="logo"
+              src="./img/nav-logo.jpg"
+              alt="website logo"
+              width="100px" />
+          </a>
+          <button class="btn-mobile-nav">
+            <ion-icon class="btn__nav btn__open" name="menu-outline"></ion-icon>
+            <ion-icon
+              class="btn__nav btn__close"
+              name="close-outline"></ion-icon>
+          </button>
+          <ul class="navigation__list">
+            <li class="navigation__list--links">
+              <a href="about.html" style="color: #e32816; border-color: #e32816"
+                >About</a
+              >
+            </li>
+            <li class="navigation__list--links"><a href="#">Home</a></li>
+            <li class="navigation__list--links"><a href="#">Join</a></li>
+            <li class="navigation__list--links"><a href="#">Sponsor</a></li>
+            <li class="navigation__list--links"><a href="#">News</a></li>
+            <button class="navtigation--btn">Campaign</button>
+          </ul>
+        </nav>
+      </header>
+
+      <!-- HOME Page Section -->
+      <section class="home__section">
+        <h1 class="home__primary--header">
+          your childrens experience <br />
+          a better eudcation
+        </h1>
+        <p class="home__description">
+          your childrens will have a better learing experience With Us
+        </p>
+        <p class="home__date">2015.10.15(THU)</p>
+        <p class="home__copy-right">@ Netional School of online</p>
+      </section>
+
+      <!-- Pragramm Section -->
+      <section class="program__section">
```

Comment:
- [ ] This section for mobile-size individual articles has its elements organized in a horizontal row, with the image at the left, the title in the center, and the description at the right 😉 

Your design

<img width="301" alt="Screen Shot 2023-02-09 at 14 10 26" src="https://user-images.githubusercontent.com/74501523/217797996-75e7246b-84e0-442b-aa0d-8eeea1e23e3a.png">

Cindy's design

<img width="452" alt="Screen Shot 2023-02-09 at 14 11 31" src="https://user-images.githubusercontent.com/74501523/217798116-06fb800f-72b5-48a0-bf54-668ceba80f04.png">